src/test/java/core/basesyntax/fileservice/CsvWriteFileServiceImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.fileservice;
+
+import core.basesyntax.errors.InvalidFileExtensionException;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvWriteFileServiceImplTest {
+    private WriteFileService csvWriteFileService = new CsvWriteFileServiceImpl();
+
+    @Test(expected = NullPointerException.class)
+    public void write_inputParameterForFileNull_notOk() {
+        csvWriteFileService.write(null, "a, banana, 30");
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void write_inputParameterForContentNull_notOk() {
+        String file = "test.csv";
```

Comment:
you should not test occurrences of NPE, you should rather throw your custom exception